style.css

```diff
@@ -102,3 +165,142 @@ main {
   align-self: center;
   margin-top: 2.5rem;
 }
+
+section {
+  margin-top: 4em;
+}
+
+.desc {
+  width: 327px;
+  height: 97px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  text-align: center;
+  margin: 26px auto;
+  color: #172b4d;
+  padding: 1em;
+  margin-top: -0.5em;
+}
+
+.divider {
+  width: 48px;
+  height: 4px;
+  background-color: #36b37e;
+  border-radius: 24px;
+  margin: 80px auto 0;
+}
+
+.cards {
+  margin: 0 20px;
+  border-radius: 5px;
+  padding-bottom: 20px;
+  border: 2px solid #ebf0ee;
+}
+
+.card {
+  text-align: center;
+}
+
+.skills {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.skills li {
+  align-items: center;
+  padding: 8px 12px;
+  background: #ebf0ee;
+  border-radius: 4px;
+  list-style-type: none;
+  margin: 5px;
+}
+
+button {
+  transition: all 1s ease;
+}
+
+.card button {
+  padding: 12px;
+  width: 122px;
+  height: 48px;
+  left: calc(50% - 122px / 2 + 0.5px);
+  bottom: 8px;
+  color: white;
+  background: #36b37f;
+  border-radius: 4px;
+}
+
+.projects {
+  display: grid;
+  grid-template-columns: 1fr;
+
+  /* grid-template-rows: repeat(auto, minmax(400px, auto)); */
+  grid-gap: 20px;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.about-me .get-my-resume {
+  padding: 12px;
+  width: 159px;
+  height: 48px;
+  background: #36b37f;
+  border-radius: 4px;
+  margin: 20px auto 40px;
+  color: white;
+}
+
+.about-me-skills .card {
+  border: 2px solid #ebf0ee;
+  margin: 0 20px;
+  border-radius: 4px;
+  background-color: #ebf0ee;
+}
+
+.about-me-skills .nested-card {
+  padding: 20px 0;
+}
+
+.nested-card button {
+  padding: 12px;
+  width: 122px;
+  height: 48px;
+  left: calc(50% - 122px / 2 + 0.5px);
+  bottom: 8px;
+  background: #36b37f;
+  border-radius: 4px;
+  color: white;
+}
```

Comment:
- [ ] Kindly make sure the typography is the same as the one on the provided Figma design.
- [ ] Kindly make sure to implement all button interactions ( `hover, pressed, etc..`) on all of your card's buttons.

    - ### YOUR DESIGN
     ![image](https://user-images.githubusercontent.com/87197412/212328914-b1b1344f-49aa-4298-909b-b90bd8dcbde7.png)

    - ### FIGMA DESIGN
    
     ![image](https://user-images.githubusercontent.com/87197412/212328989-509a3330-7962-46a9-afde-a70cbc4f854e.png)

    ![image](https://user-images.githubusercontent.com/87197412/212329424-7a8d6437-e589-4eb2-be1e-40e72b631c3a.png)